### DISCUSSION TITLE
searching the experimental HTML based front-end comes with ebusd

### DISCUSSION THREAD

**Discussion by ostauss** [john30/ebusd#668] [Q&A]
Firstly – thank you for such a great piece of soft- and hardware! I started a week ago to monitor my vaillant.
My first question is about the “experimental HTML based front-end comes with ebusd”. This is mentioned in the wiki (see chapter 7: https://github.com/john30/ebusd/wiki/7.-Integrations) 
I want to have a look at that, but I cannot find anything...
Please help :-)

Thanks in advance 
  Oliver

**Comment by john30** [comment]
you'd have to start ebusd with httpport and the right html path

**Comment by ostauss** [comment]
http port is enabled. I already use the "webservice" with path /data
Con you tell me the "path" for the html UI?

**Reply by yngndrw** [reply]
It looks like you have to add the files yourself:
https://github.com/john30/ebusd/issues/131
https://github.com/john30/ebusd/tree/master/contrib/html
